### DISCUSSION TITLE
/posts/2024/07/30-third-reconstruction-personal-blog-vitepress

### DISCUSSION THREAD

**Discussion by giscus** [Justin3go/justin3go.com#109] [Announcements]
# /posts/2024/07/30-third-reconstruction-personal-blog-vitepress

坚持深耕技术领域的T型前端程序员, 关注独立开发与AI，喜欢Vuejs、Nestjs, 还会点Python、搜索引擎、NLP、Web3、后端

https://justin3go.com/posts/2024/07/30-third-reconstruction-personal-blog-vitepress

**Comment by flashlab** [comment]
这个摘要标记有点不明显，搞得我以为内容都是AI生成的。。

**Reply by Justin3go** [reply]
确实有一点，我想想怎么修改合适一点

**Comment by feige996** [comment]
alert('unibest')
